mobile/components/AddAccount.js

```diff
@@ -0,0 +1,275 @@
+import { useState } from "react";
+import * as React from 'react'
+import { Alert, Modal, StyleSheet, Text, Pressable, View, TextInput } from "react-native";
+import axios from 'axios';
+import {useForm, Controller} from 'react-hook-form';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+
+const AddAccount = ({rerenderVaults, vaultId}) => {
+	const [state, setState] = useState('')
+	const [modalVisible, setModalVisible] = useState(false);
+	const {setValue, handleSubmit, formState: {errors, isValid}, control} = useForm({
+		defaultValues:{
+			name: '',
+            username: '',
+            password: '',
+            url: '',
+            tags: ''
+		}
+	})
+    
+    const formHandler = (values) => {
+        const selectedTags = values.tags;
+
+        if (values.url.length > 6 && !values.name && values.url.includes('.')) {
+            // if www. is in the url, set the name to the url without www.
```

Comment:
I would delete this, this is going to override user input. It is wrapped in a useEffect hook in the frontend to suggest input, not to override it.

Reply:
Gotcha. Will change